### PR TITLE
#185218925 Admin should be able to view all accounts  as well as set roles/permissions

### PR DESCRIPTION
**Pull request by cyubahiro367**
## What does this PR do?

This PR create a dashboard for the administrators to be able to manage accounts 

## Description of Task to be completed?

Admin dashboard should list all users with the option of grant permission the the user account like as seller, buyer or admin

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185218925](https://www.pivotaltracker.com/story/show/185218925)

### CODE REVIEW

**Review by Dyves1 — COMMENTED** [review]
I think you can use toast instead of snackbar

**Review by jcharles921 — CHANGES_REQUESTED** [review]
You have 2 commits , you should stash

**Review by jcharles921 — COMMENTED** [review]
another minor change for the commit message you should change it to 
`[delivers #185218925] ` for pivol tracker to link well your branch.

**Review by bergad37 — COMMENTED** [review]
Why are you using Snackbar at the same time with Toastify?

File: src/containers/Userlist.jsx
Comment (irakevine) [inline src/containers/Userlist.jsx:1]: I was wondering about this MUI
you removed it so we can all have the same codebase as team

**Review by jcharles921 — APPROVED** [review]

**Review by Gabriel-mdv — CHANGES_REQUESTED** [review]
very good job on this. However:
1. The `userList` from the containers folder was imported into `usersContainer`. It makes more sense to put userList in the components folder instead of importing it as a container into another container.  
2. changing the role of one user is showing loading effects on all role fields for all users on the page. Is there any way you can only show loading for the changed user.?

**Review by mken1205 — APPROVED** [review]

**Comment by jcharles921** [comment]
> very good job on this. However:
> 
>     1. The `userList` from the containers folder was imported into `usersContainer`. It makes more sense to put userList in the components folder instead of importing it as a container into another container.
> 
>     2. changing the role of one user is showing loading effects on all role fields for all users on the page. Is there any way you can only show loading for the changed user.?

i think it is ok to put the` userlist` as a container because it is not reused anywhere else in the code base and we also di the same thing for the landing page where we have more than one container are imported to the home container ...

**Review by nishimweprince — APPROVED** [review]
Great work @cyubahiro367 !!

**Review by irakevine — APPROVED** [review]
Great Work

**Review by Gabriel-mdv — APPROVED** [review]
pretty nice work!

**Review by bergad37 — APPROVED** [review]
Nice work!

**Pull request merged**